File: fizzbuzz.c
#include <stdio.h>

int main(int argc, const char * argv[]) {
int num;

for (num = 0; num <= 100; num++)
	if (num % 3 == 0) {
		printf("Fizz %d", num);
	}
	
	else if (num % 5 == 0) {
		printf("Buzz %d", num);
	}

	else if (num % 3 == 0 && num % 5 == 0) {
		printf("FizzBuzz %d", num);
	}

	else {
		printf("%d", num);
	}
return 0;
}

File: fizzbuzz.py
start = input("Start value: ")
end = input("End value: ")

def fizzbuzz(x):
    if x % 3 == 0 and x % 5 == 0:
        return "FizzBuzz"
    elif x % 3 == 0:
        return "Fizz"
    elif x % 5 == 0:
        return "Buzz"
    else:
        return str(x)

for x in map(fizzbuzz, range(start,end+1)):
    print "{0:>8s}".format(x)
